torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -967,7 +975,8 @@ std::shared_ptr<SugaredValue> toSugaredValue(
   } else if (py::isinstance<py::module>(obj)) {
 #ifndef USE_ROCM
     std::string obj_name = py::cast<py::str>(py::getattr(obj, "__name__"));
-    if (obj_name.compare("torch.cuda") == 0) {
+    if (obj_name.compare("torch._C") == 0 ||
```

Comment:
Yes we can. The only intention of making `torch._C.*` to avoid code duplication. I have now updated the APIs as suggested . Using `torch.jit.is_scripting` makes sure we call the right API. I'm not sure how much overhead will checking of `torch.jit.is_scripting` add?